platform/android/dependencies.gradle

```diff
@@ -21,8 +21,8 @@ ext {
             mapboxAndroidTelemetry : "com.mapbox.mapboxsdk:mapbox-android-telemetry:${mapboxServicesVersion}@aar",
 
             // mapzen lost
-            lost                   : 'com.mapzen.android:lost:3.0.0',
-
+            lost                   : 'com.mapzen.android:lost:3.0.1-SNAPSHOT',
```

Comment:
It's a fairly new feature in gradle but it should be possible: https://docs.gradle.org/2.4/release-notes.html#depending-on-a-particular-maven-snapshot-version